Smart Village App/views/content/widgets/wallCreateContentForm.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+use humhub\modules\topic\widgets\TopicPicker;
+use humhub\modules\ui\icon\widgets\Icon;
+use yii\helpers\Html;
+use humhub\modules\content\assets\ContentFormAsset;
+use humhub\modules\file\widgets\FilePreview;
+use humhub\modules\space\models\Space;
+use humhub\modules\user\widgets\UserPickerField;
+use humhub\modules\file\widgets\UploadButton;
+use humhub\modules\file\widgets\FileHandlerButtonDropdown;
+use humhub\modules\file\widgets\UploadProgress;
+use humhub\widgets\Link;
+use humhub\widgets\Button;
+
+/* @var $defaultVisibility integer */
+/* @var $submitUrl string */
+/* @var $form string */
+/* @var $submitButtonText string */
+/* @var $fileHandlers \humhub\modules\file\handler\BaseFileHandler[] */
+/* @var $canSwitchVisibility boolean */
+/* @var $contentContainer \humhub\modules\content\components\ContentContainerActiveRecord */
+
+ContentFormAsset::register($this);
+
+$this->registerJsConfig('content.form', [
+    'defaultVisibility' => $defaultVisibility,
+    'disabled' => ($contentContainer instanceof Space && $contentContainer->isArchived()),
+    'text' => [
+        'makePrivate' => Yii::t('ContentModule.base', 'Change to "Private"'),
+        'makePublic' => Yii::t('ContentModule.base', 'Change to "Public"'),
+        'info.archived' => Yii::t('ContentModule.base', 'This space is archived.')
+    ]]);
+
+$pickerUrl = ($contentContainer instanceof Space) ? $contentContainer->createUrl('/space/membership/search') : null;
+
+?>
```

Comment:
@gaurav-handysolver so this was the original part that you posted as a comment, right?

```
registerJsConfig('content.form', [ 
  'defaultVisibility' => $defaultVisibility, 
  'disabled' => ($contentContainer instanceof Space && $contentContainer->isArchived()), 
  'text' => [ 
    'makePrivate' => Yii::t('ContentModule.base', 'Change to "Private"'), 
    'makePublic' => Yii::t('ContentModule.base', 'Change to "Public"'), 
    'info.archived' => Yii::t('ContentModule.base', 'This space is archived.') 
  ]]); 

$pickerUrl = ($contentContainer instanceof Space) ? $contentContainer->createUrl('/space/membership/search') : null; 
?>
```

if you would like to post this directly on the code in the future, it would help us to talk about in a conversation instead of single comments, you know?

i formatted your string a bit to be able to compare visually. i see no changes, right?